scripts/package/truenas/changelog

```diff
@@ -1,3 +1,9 @@
+linux-6.1.49+truenas (6.1.49+truenas-1) sid; urgency=low
+
+  * Rebase local commits onto upstream v6.1.49
+
+ -- iXsystems engineering team <dev@ixsystems.com>  Mon, 29 Aug 2023 11:00:00 +0500
```

Comment:
Just a nit: "Tue, 29 Aug 2023". Same for other related PRs as well.

Reply:
Thanks, updated.